docs/StardustDocs/topics/readSqlDatabases.md

```diff
@@ -4,18 +4,33 @@ These functions allow you to interact with an SQL database using a Kotlin DataFr
 
 There are two main blocks of available functionality:
 * reading data from the database
-  * function ```readSqlTable``` reads specific database table
-  * function ```readSqlQuery``` executes SQL query
-  * function ```readResultSet``` reads from created earlier ResultSet
-  * function ```readAllSqlTables``` reads all tables (all non-system tables)
+  *  ```readSqlTable``` reads specific database table
+  *  ```readSqlQuery``` executes SQL query
+  *  ```readResultSet``` reads from created earlier ResultSet
+  *  ```readAllSqlTables``` reads all tables (all non-system tables)
 * schema retrieval
   * ```getSchemaForSqlTable``` for specific tables
   * ```getSchemaForSqlQuery``` for result of executing SQL queries
   * ```getSchemaForResultSet``` for rows reading through the given ResultSet
   * ```getSchemaForAllSqlTables``` for all non-system tables
 
-**NOTE:** This is an experimental module and for now, 
-we only support four databases: MariaDB, MySQL, PostgreSQL, and SQLite. 
+All methods above are available in the ```DataFrame``` class. 
+
+Also, there are a few **extension functions** available on Connection, 
+ResultSet, and DbConnectionConfig objects.
+
+* reading data from the database
```

Comment:
*a database. This should probably also be changed at the beginning of the file. You're not talking about a specific database (which would be "the"), just a general database, so "a" is better here :)